src/rmind/scripts/train.py

```diff
@@ -3,12 +3,13 @@
 
 import hydra
 import pytorch_lightning as pl
-import wandb
 from hydra.utils import instantiate
 from omegaconf import DictConfig, OmegaConf
 from pytorch_lightning.utilities import rank_zero_only
 from structlog import get_logger
 
+import wandb
```

Comment:
this got moved b/c it considers `wandb` a local package b/c no longer gitignored?